make/autoconf/flags-ldflags.m4

```diff
@@ -74,7 +74,11 @@ AC_DEFUN([FLAGS_SETUP_LDFLAGS_HELPER],
       # Clang needs the lld linker to work correctly
       BASIC_LDFLAGS="-fuse-ld=lld -Wl,--exclude-libs,ALL"
       if test "x$CXX_IS_USER_SUPPLIED" = xfalse && test "x$CC_IS_USER_SUPPLIED" = xfalse; then
-        UTIL_REQUIRE_PROGS(LLD, lld)
+        if test "x$TOOLCHAIN_PATH" != x; then
+          UTIL_REQUIRE_PROGS(LLD, lld, $TOOLCHAIN_PATH)
```

Comment:
No, this is not correct. Did you test this patch *without* setting a toolchain path? Doing this will remove the normal lookup in `$PATH` and only look in `$TOOLCHAIN_PATH`.

Reply:
After I remove the lld binary from `$TOOLCHAIN_PATH`, such as `mv /home/yansendao/software/acc/x86_64/bin/lld /tmp/`, the configure command generate `configure: error: Could not find required tool for LLD` as expected.

Reply:
I meant, if you have lld in your PATH but not in TOOLCHAIN_PATH and you specify a toolchain path, then your patch will introduce a regression.